<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Добавить в :</string>
   </property>
   <widget class="QRadioButton" name="radioR1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>106</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Раздел 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioR2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>106</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Раздел 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="addLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>291</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите текст :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
